mobile/lib/presentation/widgets/asset_viewer/bottom_sheet.widget.dart

```diff
@@ -140,12 +141,22 @@ class _AssetDetailBottomSheet extends ConsumerWidget {
     final exifInfo = ref.watch(currentAssetExifProvider).valueOrNull;
     final cameraTitle = _getCameraInfoTitle(exifInfo);
 
+    Future<void> editDateTime() async {
+      await ref.read(actionProvider.notifier).editDateTime(ActionSource.viewer, context);
+    }
+
     return SliverList.list(
       children: [
         // Asset Date and Time
         _SheetTile(
           title: _getDateTime(context, asset),
           titleStyle: context.textTheme.bodyMedium?.copyWith(fontWeight: FontWeight.w600),
+          trailing: IconButton(
+            icon: const Icon(Icons.edit, size: 18),
```

Comment:
There is an additional padding around the IconButton by default. Can you add this to remove it so the edit icon is properly right aligned?
```suggestion
            constraints: const BoxConstraints(),
            padding: EdgeInsets.zero,
            style: const ButtonStyle(
              tapTargetSize: MaterialTapTargetSize.shrinkWrap, // the '2023' part
            ),
            icon: const Icon(Icons.edit, size: 18),
```